packages/wallet/src/redux/protocols/advance-channel/__tests__/scenarios.ts

```diff
@@ -53,6 +47,18 @@ const propsB = {
   ourIndex: ThreePartyPlayerIndex.B,
 };
 
+const propsHub = {
+  ...props,
+  ourIndex: ThreePartyPlayerIndex.Hub,
+};
+
+const commitments0 = [signedCommitment0];
```

Comment:
Do you think there is a more descriptive yet succinct variable name that describes the channel state for the sequence of commitments?

Reply:
Perhaps, but I'm far ahead of origin's HEAD locally, so I would prefer not to rename it in this PR.